vyper/compiler/__init__.py

```diff
@@ -121,10 +128,18 @@ def outputs_from_compiler_data(
         output_formats = ("bytecode",)
 
     ret = {}
+
     with anchor_settings(compiler_data.settings):
         for output_format in output_formats:
             if output_format not in OUTPUT_FORMATS:
                 raise ValueError(f"Unsupported format type {repr(output_format)}")
+
+            is_vyi = compiler_data.file_input.resolved_path.suffix == ".vyi"
```

Comment:
technically we can just inspect `compiler_data.compilation_target.is_interface` now, but i think this is also ok for clarity